server/src/main/java/org/apache/iotdb/db/conf/IoTDBDescriptor.java

```diff
@@ -309,6 +309,11 @@ private void loadProps() {
       conf.setMemtableNumInEachStorageGroup(
           Integer.parseInt(properties.getProperty("memtable_num_in_each_storage_group", String.valueOf(conf.getMemtableNumInEachStorageGroup()))));
 
+      // the num of memtables in each storage group
```

Comment:
what's this comment mean...

Reply:
Fixed. Sorry for misunderstanding. This should be "the default fill interval in LinearFill and PreviousFill".